server/config/firebaseInit.js

```diff
@@ -0,0 +1,15 @@
+// import admin from 'firebase-admin';
+
+const admin = require('firebase-admin');
+
+const serviceAccount = require('./firebase.json');
```

Comment:
Will need to place this file at root and change the require path for Render secret file usage:

![image](https://github.com/awctw/study-dash/assets/56493743/2804dce3-0618-4f88-acac-3c2146e620b9)